ugvc/mrd/srsnv_plotting_utils.py

```diff
@@ -15,10 +15,28 @@
 from sklearn.metrics import confusion_matrix
 from tqdm import tqdm
 
+from ugvc import logger
 from ugvc.srsnv.srsnv_utils import plot_precision_recall, precision_recall_curve
 from ugvc.utils.metrics_utils import read_effective_coverage_from_sorter_json
 from ugvc.utils.misc_utils import exec_command_list
 
+default_LoD_filters = {
+    "no_filter": "X_SCORE >= 0",
+    "edit_distance_5": "X_SCORE >= 0 and X_EDIST <= 5",
+    "BQ60": "X_SCORE >= 6 and X_EDIST <= 5",
+    "BQ80": "X_SCORE >= 7.9 and X_EDIST <= 5",
+    "CSKP": "X_SCORE >= 10 and X_EDIST <= 5",
+    "BQ80_trim_ends": "X_SCORE >= 7.9 and X_EDIST <= 5 and X_INDEX > 12 and X_INDEX < (X_LENGTH - 12)",
+    "CSKP_trim_ends": "X_SCORE >= 10 and X_EDIST <= 5 and X_INDEX > 12 and X_INDEX < (X_LENGTH - 12)",
+    "BQ60_mixed_only": "X_SCORE >= 6 and X_EDIST <= 5 and is_mixed",
+    "BQ80_mixed_only": "X_SCORE >= 7.9 and X_EDIST <= 5 and is_mixed",
+    "CSKP_mixed_only": "X_SCORE >= 10 and X_EDIST <= 5 and is_mixed",
+    "BQ80_trim_ends_mixed_only": "X_SCORE >= 7.9 and X_EDIST <= 5 and X_INDEX > 12 \
+        and X_INDEX < (X_LENGTH - 12) and is_mixed",
+    "CSKP_trim_ends_mixed_only": "X_SCORE >= 10 and X_EDIST <= 5 and X_INDEX > 12 \
+        and X_INDEX < (X_LENGTH - 12) and is_mixed",
+}
+
 
 def create_data_for_report(
     xgb_classifier: xgb.XGBClassifier,
```

Comment:
In the rest of the code the classifier is generalized to be an sklearn classifier and not necessarily xgboost, is there anything XGBoost specific here? If not please change to just "classifier"

Reply:
done